src/core/utils/index.js

```diff
@@ -77,10 +77,10 @@ export function isValidClientApp(value, { _config = config } = {}) {
   return _config.get('validClientApplications').includes(value);
 }
 
-export function sanitizeHTML(text, allowTags = []) {
+export function sanitizeHTML(text, allowTags = [], _purify = purify()) {
```

Comment:
This will create a new instance for every call. 

I'm not sure how heavy this is but we might want to be sure this doesn't cause a perf issue.

Reply:
indeed 😞 

So I guess I have no choice but duplicating the creation of the purify instance.